generator/build.gradle

```diff
@@ -0,0 +1,54 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+dependencies {
+    compile project(':schemas')
+    compile 'com.google.inject:guice:4.1.0'
+    compile 'dk.brics.automaton:automaton:1.11-8'
+    compile 'com.fasterxml.jackson.core:jackson-core:2.9.6'
+    compile 'com.fasterxml.jackson.core:jackson-annotations:2.9.6'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.9.6'
+    compile 'org.apache.commons:commons-csv:1.5'
+    testCompile 'org.junit.jupiter:junit-jupiter-api:5.3.1'
+    testCompile 'io.cucumber:cucumber-core:4.0.0'
+    testCompile 'io.cucumber:cucumber-junit:4.0.0'
+    testCompile 'io.cucumber:cucumber-java:4.0.0'
+    testCompile 'io.cucumber:cucumber-jvm:4.0.0'
+    testCompile 'io.cucumber:gherkin:5.0.0'
+    testCompile 'io.cucumber:cucumber-expressions:6.0.1'
+    testCompile 'info.cukes:cucumber-picocontainer:1.2.5'
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.junit.platform:junit-platform-runner:1.3.1'
+    testCompile 'org.junit.vintage:junit-vintage-engine:5.3.1'
+    testCompile 'org.hamcrest:java-hamcrest:2.0.0.0'
+    testCompile 'org.junit.jupiter:junit-jupiter-params:5.2.0'
+    testCompile 'com.shazam:shazamcrest:0.11'
+    testCompile 'org.mockito:mockito-all:1.9.5'
+    testCompile 'org.junit.jupiter:junit-jupiter-engine:5.3.1'
+    testImplementation('org.junit.jupiter:junit-jupiter:5.4.2')
+}
+
+test {
+    useJUnitPlatform()
+}
+
+description = 'DataHelix Generator'
+
+task release(type: Jar) {
+	manifest {
+		attributes 'Created-By': 'Scott Logic',
+			'Implementation-Version': version != null ? version : "0.0",
+			'Built-By': System.properties['user.name'],
```

Comment:
Larger question, should we include windows usernames in the build artefact?

Reply:
Just had a chat with @sl-slaing about this.  The overall points were:

* this is a separate issue from converting to gradle
* because of that, we should not make the change in this PR
* Taking this out now would ensure it is not forgotten, so we should do that, but under a separate ticket and PR
* We should find out what best practice is among the wider Java community for setting this attribute

Reply:
Removed for the time being after discussions.